Description: Bump bounds for newer semigroupoids
 Bump bounds for semigroupoids 5.2, shipped in unstable
Author: Ilias Tsitsimpis <iliastsi@debian.org>
Forwarded: not-needed, fixed in newer version
Applied-Upstream: https://github.com/diagrams/monoid-extras/commit/3aee8cb1c072b
Index: b/monoid-extras.cabal
===================================================================
--- a/monoid-extras.cabal
+++ b/monoid-extras.cabal
@@ -38,7 +38,7 @@ library
   build-depends:     base >= 4.3 && < 4.10,
                      groups < 0.5,
                      semigroups >= 0.8 && < 0.19,
-                     semigroupoids >= 4.0 && < 5.2
+                     semigroupoids >= 4.0 && < 5.3
 
   hs-source-dirs:    src
 
